src/components/image/image.scss

```diff
@@ -1,5 +1,7 @@
 .flex-image {
-  width: 100%;
-  height: 100%;
-  object-fit: contain;
+  max-width: 100%;
+  max-height: 100%;
+  width: 100vw;
+  height: 100vw;
```

Comment:
I have some changes to this class. Wouldn't making the width 100vw means we can't use FlexImage to show 3 in a row?

Reply:
Yup but there is the `object-fit: contain` property so it will squeeze and fit into the parent container while maintaining aspect ratio.